### ISSUE TITLE
update docs

### ISSUE THREAD

**Issue by nofear-js** [greenheartgames/greenworks#279]
u need to update docs, because the method names in docs do not match the real declared methods

https://github.com/greenheartgames/greenworks/blob/master/docs/friends.md
`greenworks.getFriendsAccount(friend_flag)` != `SET_FUNCTION("getFriendCount", GetFriendCount);`

`getFriendsAccount` change to `getFriendCount`

because of such a trifle, i spent 10 minutes to understand why my application knocks out an error 
`TypeError: greenworks_1.default.geFriendsAccount is not a function` :(

Hopefully in other methods everything is fine!

**Comment by qq253498229** [comment]
Yes, now before calling the api, we should print greenworks object to confirm whether there is the api we want.
